packages/frontend/src/app/page.tsx

```diff
@@ -16,24 +19,24 @@ import ChatsList from "../components/ChatsList";
 import MessagingPanel from "../components/MessagingPanel";
 
 export default function Home() {
-	const [selectedChat, setSelectedChat] = useState(
-		"Choose a friend to get started!"
-	);
-
-	useEffect(() => {
-		const getData = async () => {
-			const data = await fetch("http://localhost:8000/");
-			const jsonResponse = await data.json();
-			console.log(jsonResponse);
-		};
-
-		getData();
-	}, []);
-
-	return (
-		<div className="flex w-screen h-screen">
-			<ChatsList onChatSelect={setSelectedChat} />
-			<MessagingPanel selectedFriend={selectedChat} />
-		</div>
-	);
+  const [selectedChat, setSelectedChat] = useState(
+    "Choose a friend to get started!"
+  );
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = await fetch("http://localhost:8000/");
+      const jsonResponse = await data.json();
+      console.log(jsonResponse);
+    };
+
+    getData();
+  }, []);
+
+  return (
+    <div className="flex w-screen h-screen">
+      <ChatsList onChatSelect={setSelectedChat} />
+      <MessagingPanel selectedFriend={selectedChat} />
+    </div>
+  );
```

Comment:
This was auto-formatted, I hope this is the last time we deal with this; We can update the settings if you prefer to have them wider.